app/src/main/java/com/infomaniak/mail/ui/main/menu/MenuDrawerFragment.kt

```diff
@@ -153,7 +153,7 @@ class MenuDrawerFragment : MenuFoldersFragment(), MailboxListFragment {
         }
 
         syncAutoConfig.setOnClickListener {
-            startActivity(Intent(context, SyncAutoConfigActivity::class.java))
+            (activity as MainActivity).navigateToSyncAutoConfigActivity()
```

Comment:
What if we create a 'launchSyncAutoConfigActivity' fonction in Extension.kt along with all the other launchActivity functions, so you won't have to cast both times